frontend/lib/service-definitions/service-definition-prop-types.js

```diff
@@ -10,18 +10,20 @@ const objectOfKeyValuesPropType = PropTypes.objectOf(PropTypes.string)
 
 const examplePropType = PropTypes.exact({
   title: PropTypes.string.isRequired,
+  link: PropTypes.string,
   example: PropTypes.exact({
     pattern: PropTypes.string.isRequired,
     namedParams: objectOfKeyValuesPropType,
     queryParams: objectOfKeyValuesPropType,
   }).isRequired,
   preview: PropTypes.exact({
     label: PropTypes.string,
-    message: PropTypes.string.isRequired,
-    color: PropTypes.string.isRequired,
+    message: PropTypes.string,
+    color: PropTypes.string,
     style: PropTypes.string,
     namedLogo: PropTypes.string,
-  }).isRequired,
+    buildFromExample: PropTypes.bool,
+  }),
```

Comment:
The goal of this schema is that it's kept in sync with `core/base-service/service-definitions.js`. Would it be possible to create a new schema that for `<BadgeExamples />`? You could refactor this one if you want to reuse part of it.

Reply:
Thanks for this suggestion. This part of code wasn't perfect and I was hoping for your ideas. I created a separate schema.